src/functions/module_statistical.ts

```diff
@@ -80,7 +81,8 @@ function covariance(dataY: Arg, dataX: Arg, isSample: boolean): number {
 
   assert(
     () => count !== 0 && (!isSample || count !== 1),
-    _t("Evaluation of function [[FUNCTION_NAME]] caused a divide by zero error.")
+    _t("Evaluation of function [[FUNCTION_NAME]] caused a divide by zero error."),
```

Comment:
is the condition "(!isSample || count !== 1)" really a divide by zero error ?

Reply:
:man_shrugging: not really but Google and/or Excel also behaves that way (I don't remember if I tried one or the other or both)